### PR TITLE
documented label cluster-service

### PR DESCRIPTION
**Pull request by kundan2707**
### Description
Documented label cluster-service

### Issue
#48609 

Closes: #

### CODE REVIEW

**Comment by kundan2707** [comment]
[Preview](https://deploy-preview-48988--kubernetes-io-main-staging.netlify.app/docs/reference/labels-annotations-taints/)

File: content/en/docs/reference/labels-annotations-taints/_index.md [hunk 1]
```diff
@@ -1423,6 +1423,17 @@ Used on: Ingress
 Starting in v1.18, this annotation is deprecated in favor of `spec.ingressClassName`.
 {{< /note >}}
 
+### kubernetes.io/cluster-service (deprecated)
```
Comment (sftim) [inline content/en/docs/reference/labels-annotations-taints/_index.md:1426]:
```suggestion
### kubernetes.io/cluster-service (deprecated) {#kubernetes-io-cluster-service}
```

File: content/en/docs/reference/labels-annotations-taints/_index.md [hunk 2]
```diff
@@ -1423,6 +1423,17 @@ Used on: Ingress
...
+
+Type: Label
+
+Example: kubernetes.io/cluster-service: "true"
+
+Used on: Service
+
+Addon manager deprecates this label for usage in addon manager. It is not deprecated in general.
+kubectl cluster-info depends on it and this annotation appear there.
```
Comment (sftim) [inline content/en/docs/reference/labels-annotations-taints/_index.md:1435]:
```suggestion
This label indicates that the Service provides a service to the cluster, if the value is set to true.
When you run `kubectl cluster-info`, the tool queries for Services with this label set to true.

However, setting this label on any Service is deprecated.
```
I wouldn't mention addon manager here.

**Review by sftim — COMMENTED** [review]
/lgtm

How about these tweaks?

File: content/en/docs/reference/labels-annotations-taints/_index.md (hunk 1 above)
Comment (kundan2707) [reply content/en/docs/reference/labels-annotations-taints/_index.md]: made changes as suggested

File: content/en/docs/reference/labels-annotations-taints/_index.md (hunk 2 above)
Comment (kundan2707) [reply content/en/docs/reference/labels-annotations-taints/_index.md]: made changes as suggested

File: content/en/docs/reference/labels-annotations-taints/_index.md [hunk 3]
```diff
@@ -1423,6 +1423,19 @@ Used on: Ingress
 Starting in v1.18, this annotation is deprecated in favor of `spec.ingressClassName`.
 {{< /note >}}
 
+### kubernetes.io/cluster-service (deprecated) {#kubernetes-io-cluster-service}
+
+Type: Label
+
+Example: kubernetes.io/cluster-service: "true"
```
Comment (dipesh-rawat) [inline content/en/docs/reference/labels-annotations-taints/_index.md:1430]: _Minor non-blocking nit_: Adjust the formatting of the example block to match the style of all the other examples on this [page](https://kubernetes.io/docs/reference/labels-annotations-taints/).

```suggestion
Example: `kubernetes.io/cluster-service: "true"`
```

Comment (kundan2707) [reply content/en/docs/reference/labels-annotations-taints/_index.md]: @dipesh-rawat incorporated suggestion.

**Review by dipesh-rawat — COMMENTED** [review]
/lgtm

**Comment by sftim** [comment]
/approve

/hold

OK to unhold once this passes **all** tests. The hold is just to prevent a flood of merges after v1.32 is released.

**Comment by dipesh-rawat** [comment]
/unhold

v1.32 has been released and website has been thawed. 
Also, a merge of PR will re-trigger website build to help with refresh of binaries on download page.

**Pull request merged**